include/match.h

```diff
@@ -35,7 +35,7 @@
 
 /** Information about a matched component. */
 typedef enum xcb_xrm_match_flags_t {
-    MT_NONE = 1 << 0,
+    MF_NONE = 1 << 0,
```

Comment:
Yes! :) Although I'm just now thinking that maybe there should be a function that does the match on a single component and returns the match flags. Then we can avoid the duplication of "did it match?" we have now – and then we'd also actually need this bit.